src/System/Wlog/Wrapper.hs

```diff
@@ -40,10 +40,11 @@ import           System.Wlog.Severity       (Severity (..))
 streamHandlerWithLock :: (Handle -> Text -> IO ())
                       -> MVar ()
                       -> Handle
+                      -> (Handle -> Bool)
```

Comment:
Could you please rename `Wrapper` module to `Terminal` module? `Wrapper` is really odd name... It's such a name only by historical reasons. Would you mind perform a little bit extra refactoring? :)

Reply:
Of course, it's more suitable name :slightly_smiling_face: